sparse/rest/server.go

```diff
@@ -0,0 +1,27 @@
+package rest
+
+import (
+	"fmt"
+	"net/http"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancher/sparse-tools/sparse"
+)
+
+type SyncServer struct {
+	filePath      string
+	holeIntervals []sparse.Interval
```

Comment:
Just to clarify you can keep the fileIo reference.  Just the layout info doesn't seem needed.